build.gradle

```diff
@@ -86,11 +86,11 @@ subprojects {
             project.android.dexOptions.preDexLibraries = rootProject.ext.preDexLibs
         }
     }
-    project.plugins.apply('net.ltgt.errorprone')
-
-    configurations.errorprone {
-        resolutionStrategy.force 'com.google.errorprone:error_prone_core:2.0.15'
-    }
+//    project.plugins.apply('net.ltgt.errorprone')
```

Comment:
@digitalbuddha That's probably a mistake, since the project wouldn't run with error-prone enabled.